tests/integration_testing/beetl/sources/test_mongodb.py

```diff
@@ -184,3 +209,20 @@ def test_sync_between_mongodb_and_mysql(self):
 
                 oneRecordWasDeleted = ManualResult(0, 0, 1)
                 self.assertEqual(deleteResult, oneRecordWasDeleted)
+
+    def test_store_diff__when_specified_in_sync__diff_is_stored_in_the_specified_collection(
+        self,
+    ):
+        with MongoDbContainer() as mongodb:
+            connection_string = mongodb.get_connection_url()
+            config = diff_to_mongodb(connection_string)
+            beetl_instance = beetl.Beetl(beetl.BeetlConfig(config))
+
+            # Act
+            beetl_instance.sync()
+
+            mongo_client = MongoClient(connection_string)
+            db = mongo_client[DATABASE_NAME]
+            collection = db["diff"]
+            result = collection.find_one()
+            self.assertIsNotNone(result)
```

Comment:
Test content of read data.

Reply:
Done.
Also changed so that the nested dicts are stored as such and not json strings.